docs/release-instruction.md

```diff
@@ -0,0 +1,56 @@
+# Release instructions
+
+## Pre-release checks
+
+These should run successfully:
+ * build,
+ * unit-tests,
+ * integration tests in 
+ [neofs-devenv](https://github.com/nspcc-dev/neofs-devenv).
+
+## Writing changelog
+
+Add an entry to the `CHANGELOG.md` following the style established there. Add an 
+optional codename, version and release date in the heading. Write a paragraph
+describing the most significant changes done in this release. Add
+`Fixed`, `Added`, `Removed` and `Updated` sections with fixed bug descriptions
+and changes. Describe each change in detail with a reference to github issues if
+possible. 
+
+## Tag the release
+
+Use `vX.Y.Z` tag following the [semantic versioning](https://semver.org/) 
```

Comment:
Maybe we should a note from @im-kulikov regarding the RC.X tag format.

Proposed format: vX.Y.Z-rc.N

Reply:
Mentioned `vX.Y.Z-rc.N`.